src/client/proxy/handlers/storage.ts

```diff
@@ -2,9 +2,11 @@ import {FastifyInstance, FastifyRequest} from 'fastify';
 const {uploadFile, getFile, FILE_TYPE} = require('../../storage');
 // @ts-expect-error no types for package
 import detectContentType from 'detect-content-type';
+import isSVG from 'is-svg';
```

Comment:
I was trying to avoid new dependencies like this one. But when getting content from storage, where we don't have file extensions available, checking if something is an SVG has some corner-cases that I think justify using a third-party library. I went through the test cases in `is-svg` and they look like a good fit for our use-case.

`detectContentType` unfortunately does not properly recognize SVG, it returns _text/xml_ or _text/plain_.